src/main/java/com/rackspace/ceres/app/services/QueryService.java

```diff
@@ -124,6 +130,55 @@ public QueryService(ReactiveCqlTemplate cqlTemplate,
         .checkpoint();
   }
 
+  public Flux<TsdbQueryResult> queryTsdbDownsampled(String tenant, List<TsdbQueryRequest> queries,
```

Comment:
Can you rename this to something like `queryTsdb` as this method is doing processing for both `raw` and `downsampled` data.

Reply:
done